backend/sportsmatch/src/test/java/com/sportsmatch/services/PlaceServiceTest.java

```diff
@@ -0,0 +1,111 @@
+package com.sportsmatch.services;
+
+import com.sportsmatch.dtos.PlaceDTO;
+import com.sportsmatch.mappers.PlaceMapper;
+import com.sportsmatch.models.Place;
+import com.sportsmatch.repositories.PlaceRepository;
+import org.junit.jupiter.api.Test;
+import org.mockito.InjectMocks;
+import org.mockito.Mock;
+import org.springframework.boot.test.context.SpringBootTest;
+import org.springframework.http.HttpStatus;
+import org.springframework.http.ResponseEntity;
+
+import java.util.ArrayList;
+import java.util.List;
+
+import static org.junit.jupiter.api.Assertions.assertEquals;
+import static org.mockito.Mockito.*;
+
+@SpringBootTest
+public class PlaceServiceTest {
+  @Mock
+  private PlaceRepository placeRepository;
+  @Mock
+  private PlaceMapper placeMapper;
+  @InjectMocks
+  private PlaceService placeService;
+
+  Place createPlaceEntity() {
+    return Place.builder()
+        .name("Test Place")
+        .address("Test Address")
+        .latitude(123.456F)
+        .longitude((789.012F))
+        .build();
+  }
+
+  PlaceDTO createPlaceDTO() {
+    return PlaceDTO.builder()
+        .name("Test Place")
+        .address("Test Address")
+        .latitude(123.456F)
+        .longitude(789.012F)
+        .build();
+  }
+
+  @Test
+  public void AddNewPlaceShouldReturnSuccessMessage() {
```

Comment:
checkstyle error: method name should begin with lowercase
![image](https://github.com/MatejFrnka/sportsmatch/assets/131284389/683127ca-df4c-4b66-8e7f-107483381ab1)